packages/ui5-application-writer/test/data.test.ts

```diff
@@ -292,31 +305,46 @@ describe('Setting defaults', () => {
             version: '0.0.1'
         } as App;
 
-        expect(mergeApp(app)).toEqual(expectedApp);
+        test('minimal input', async () => {
+            expect(await mergeApp(baseInput)).toEqual(expectedApp);
+        });
 
-        // Test toolsId is correctly set, sourceTemplate.id and sourceTemplate.version are set if not provided.
-        const toolsId = 'guid:abcd1234';
-        app.sourceTemplate = {
-            toolsId
-        };
-        expectedApp.sourceTemplate = {
-            id: '',
-            toolsId,
-            version: ''
-        };
-        expect(mergeApp(app)).toEqual(expectedApp);
+        test('toolsId provided but not id/version of source template', async () => {
+            const toolsId = 'guid:abcd1234';
+            const merged = await mergeApp({
+                ...baseInput,
+                sourceTemplate: {
+                    toolsId
+                }
+            });
+            expect(merged.sourceTemplate).toEqual({ ...expectedApp.sourceTemplate, toolsId });
+        });
 
-        // Test toolsId is correctly set, sourceTemplate.id and sourceTemplate.version are set to provided value.
-        app.sourceTemplate = {
-            id: 'test-source-template-id',
-            version: '9.9.9',
-            toolsId
-        };
-        expectedApp.sourceTemplate = {
-            id: 'test-source-template-id',
-            version: '9.9.9',
-            toolsId
-        };
-        expect(mergeApp(app)).toEqual(expectedApp);
+        test('source template provided', async () => {
+            const sourceTemplate = {
+                id: 'test-source-template-id',
+                version: '9.9.9',
+                toolsId: 'guid:abcd1234'
+            };
+            const merged = await mergeApp({
+                ...baseInput,
+                sourceTemplate
+            });
+            expect(merged.sourceTemplate).toEqual(sourceTemplate);
+        });
+    });
+
+    describe('getSpecTagVersion', () => {
+        test('get latest version', async () => {
+            expect(await getSpecTagVersion('')).toBe('latest');
```

Comment:
await can be removed as it is no longer async

Reply:
done